group-coordinator/src/main/java/org/apache/kafka/coordinator/group/runtime/CoordinatorPlayback.java

```diff
@@ -33,4 +33,18 @@ public interface CoordinatorPlayback<U> {
      * @throws RuntimeException if the record can not be applied.
      */
     void replay(U record) throws RuntimeException;
+
+    /**
+     * Invoke operations when a batch has been successfully loaded.
+     *
+     * @param offset the offset of the last record in the batch.
+     */
+    void onBatchLoaded(Long offset);
```

Comment:
Should we align the naming on the method that we have in the coordinator context? It is kind of confusing if we use different terms here, I think.